routes/votings/list-of-votings-api.js

```diff
@@ -25,8 +26,9 @@ router.get("/:idOfUser", async (req, res) => {
 
 //insertion of new voting record
 router.post("/", async (req, res) => {
-    const {error} = validateUser(req.body)
-    if(error) return res.status(400).send(error.details[0].message)
+    console.log(req.body);
+    // const {error} = validateUser(req.body)
+    // if(error) return res.status(400).send(error.details[0].message)
```

Comment:
Have you forgot to uncomment this? Because it is actually a good thing to respond with _400 Bad Request_ when the request body is not valid.

Or another option would be to delegate the validation to the database and wrap the `db.createList(req.body)` expression in a try-catch and if the error is about invalid data-structure, then respond with _400 Bad Request_.  But it's probably best to have the schema of the API defined here in the code so that the application is not tightly coupled to a concrete database implementation.

Reply:
there are not such comments in the file.